content/02.main-text.md

```diff
@@ -151,6 +151,18 @@ For example, figures can be specified using versioned URLs that refer to the cod
 In addition, manuscripts can be templated, so that numerical values or tables get inserted directly from the repository that created them.
 An [example repository](https://github.com/greenelab/manubot-rootstock) demonstrates Manubot's features and serves as a template for users to write their own manuscript with Manubot.
 
+Since its creation to facilitate the Deep Review, Manubot has been used to write a variety of scholarly documents.
+The [Sci-Hub Coverage Study](https://github.com/greenelab/scihub-manuscript) --- performed openly on GitHub from its inception --- investigated Sci-Hub's repository of pirated articles [@doi:10.7554/eLife.32822].
+Sci-Hub [reviewed](https://github.com/greenelab/scihub-manuscript/issues/17) the initial preprint from this study in a series of tweets, pointing out a major error in one of the analyses.
+Within hours, the authors used markdown's strikethrough formatting in Manubot to cross-out the errant sentences ([commit](https://github.com/greenelab/scihub-manuscript/commit/8fcd0cd665f6fb5f39bed7e26b940aa27d4770ba), [versioned manuscript](https://greenelab.github.io/scihub-manuscript/v/8fcd0cd665f6fb5f39bed7e26b940aa27d4770ba/)), thereby alerting readers to the mistake and preventing further propagation of misinformation.
+A larger [set of revisions](https://github.com/greenelab/scihub-manuscript/pull/19) that was merged a month later explained the error in more detail, and was included in a second version of the preprint.
+As such, continuous publishing via Manubot helped the authors address the error without delay, while retaining a public version history of the process.
+Of interest, in 2017, the Sci-Hub Coverage Study preprint was the [most viewed](http://web.archive.org/web/20171221221858/http://www.prepubmed.org/top_preprints/) _PeerJ Preprint_, while the Deep Review was the most viewed _bioRxiv_ preprint [@doi:10.1038/d41586-017-08493-x].
+Hence, in the Manubot's first year, its collaborative, open, and review-driven authoring process yielded two of the most popular preprints.
+
+Additional research studies in progress are being authored using Manubot, including in the fields of [genomics](https://vsmalladi.github.io/tfsee-manuscript/), [climate science](https://openclimatedata.github.io/global-emissions/), and [data visualization](https://yt-project.github.io/yt-3.0-paper/).
+Manubot is also being used for documents beyond traditional journal publications, such as [grant proposals](https://greenelab.github.io/manufund-2018/), [progress reports](https://greenelab.github.io/czi-hca-report/), [undergraduate research reports](https://zietzm.github.io/Vagelos2017/) [@doi:10.6084/m9.figshare.5346577], [literature reviews](https://slochower.github.io/synthetic-motor-literature/), and lab notebooks.
+Finally, manuscripts written with other authoring systems have been successfully ported to Manubot, including the Bitcoin Whitepaper [@tag:steemit-post] and Project Rephetio manuscript [@doi:10.7554/eLife.26726].
```

Comment:
Was using the computing definition:

> transfer (software) from one system or machine to another.

I just mean taken from an existing authoring program and converted into a Manubot manuscript. I think this is clear based on the sentence context, but am open to suggested alternative wordings if anyone feels strongly.

Reply:
I know that meaning. But you are just making the point that you moved the content over, right? The sentence is a little ambiguous to me because it sorta sounds like "other authoring systems have been ported." Maybe it's totally fine as is. I probably just read it too quickly the first time.

Reply:
Would `successfully recreated with Manubot` be less ambiguous?  Porting made sense to me, but we may have non-technical readers.

Reply:
recreated is okay, but it almost implies some of the original authoring process was repeated which was not the case. Unless there is major confusion here, I'm in favor of proceeding with porting.